x-pack/plugins/search_indices/public/hooks/api/use_index_mappings.ts

```diff
@@ -8,12 +8,16 @@
 import { useQuery } from '@tanstack/react-query';
 import { useKibana } from '../use_kibana';
 import { Mappings } from '../../types';
+import { QueryKeys } from '../../constants';
 
+const POLLING_INTERVAL = 15 * 1000;
```

Comment:
Nit: Could we move this polling interval to a constants file as well ? 
I think we are using same polling interval in other hooks too. https://github.com/elastic/kibana/blob/3ce8b74c48a0bbee3fcabf6ec9fcae8ab1c88dcc/x-pack/plugins/search_indices/public/hooks/api/use_index.ts#L13 https://github.com/elastic/kibana/blob/3ce8b74c48a0bbee3fcabf6ec9fcae8ab1c88dcc/x-pack/plugins/search_indices/public/hooks/api/use_indices_status.ts#L16